NuRadioMC/SignalProp/directRayTracing.py

```diff
@@ -48,13 +48,13 @@ def get_path_length(self, iS):
     def get_travel_time(self, iS):
         traveltime = 0
         path = self.get_path(iS)
-        segment = [path[0][1]-path[0][0],path[1][1]-path[1][0],path[2][1]-path[2][0]]
+        segment = [path[1][0]-path[0][0],path[1][1]-path[0][1],path[1][2]-path[0][2]]
         r = np.linalg.norm(segment)
-        for i in range(len(path[0])-1):
-            xx = np.array([path[0][i], path[1][i], path[2][i]])
-            yy = np.array([path[0][i+1], path[1][i+1], path[2][i+1]])
-            x = (xx+yy)/2
-            n = self._medium.get_index_of_refraction(x)
+        for i in range(len(path)-1):
+            xx = np.array([path[i][0], path[i][1], path[i][2]])
+            yy = np.array([path[i+1][0], path[i+1][1], path[i+1][2]])
+            center_of_segment = (xx+yy)/2
+            n = self._medium.get_index_of_refraction(center_of_segment)
```

Comment:
I think it is always nice to have the iterator variables give some indication of what they are iterating over.
so like ieng or isol or icos. Whatever is appicable here

Reply:
I tried to keep the changes minimal to only cover the bug. There are a few things I would like to change. Maybe I should just do it.

Reply:
I think there is some precident to update the code to make it more readable too.

Maybe change `i` to `istep`.

I really don't see why this code does the casing to an array when you could just do `xx = path[istep]` and `yy=path[istep+1]`

I guess we can leave it for now.

Reply:
Exactly. I will push a more readable version of the code now.

Reply:
I got rid of the loop too.

Reply:
I realize that no one really uses this, but this should actually make the code a lot faster :D